packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -614,6 +615,12 @@ export interface CallCompositeStrings {
     cameraOffBackgroundEffectWarningText?: string;
     cameraPermissionDenied: string;
     cameraTurnedOff: string;
+    capabilityShareScreenGrantedDueToRoleChangeToPresenter?: string;
+    capabilityShareScreenLostDueToRoleChangeToAttendee?: string;
+    capabilityTurnVideoOnGrantedDueToMeetingOption?: string;
+    capabilityTurnVideoOnLostDueToMeetingOption?: string;
```

Comment:
nit: perhaps `meeting` -> `call` for these (and maybe option -> capability?)
```suggestion
    capabilityTurnVideoOnLostDueToCallCapability?: string;
```

Reply:
Alot of these capabilities are aligned to match the capability being set in Teams. I believe in Teams they be using the "meeting" terminology which can be confusing. Aligning to similar wording could be helpful for those having to navigate from the Teams side. @mgamis-msft  can you share with us how we are coming up with these labels and from where were we inspired to use these labels?

Reply:
I chose MeetingOption because the reasons capabilities can change is defined in the Calling SDK API below:
```
/**
 * Capabilities Changed Reason
 * @public
 */
export declare type CapabilitiesChangedReason = 'RoleChanged' | 'UserPolicyChanged' | 'MeetingOptionOrOrganizerPolicyChanged';
```
reference: https://www.npmjs.com/package/@azure/communication-calling/v/1.16.3-beta.1?activeTab=code

I thought adding the reason to the string key would be more specific since we probably we another string if turnVideoOn is lost through another reason. I also though 'capabilityTurnVideoOnLostDueToMeetingOptionOrOrganizerPolicy' is too long unless you think it is okay.

Reply:
As of right now, a capability change with the reason `UserPolicyChanged` is not something that is possible because user policies can only be applied to Teams users. That could change in the future but if we were to receive this reason then we would not show a notification.